### PR TITLE
Bump xamlTools to 17.13.35611.21

### PR DESCRIPTION
**Pull request by kunyli**
Bump xamlTools to 17.13.35611.21

### CODE REVIEW

**Review by etvorun — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -43,7 +43,7 @@
   * When converting to raw strings, do not change \r\n sequences if not explicitly requested by the user (PR: [#76120](https://github.com/dotnet/roslyn/pull/76120))
   * Exclude starting symbol when going to implementations if we find implementations (PR: [#76125](https://github.com/dotnet/roslyn/pull/76125))
 * Add option to disable auto insertion of documentation comments (PR: [#7381](https://github.com/dotnet/vscode-csharp/pull/7381))
-* Bump xamltools to 17.13.35606.23 (PR: [#7856](https://github.com/dotnet/vscode-csharp/pull/7856))
+* Bump xamltools to 17.13.35611.21 (PR: [#7872](https://github.com/dotnet/vscode-csharp/pull/7872))
```
Comment (JoeRobich) [inline CHANGELOG.md:46]: We have already snapped for the 2.60 prerelease. Can you revert this line and add a new one to the 2.61.x section?

**Review by JoeRobich — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (kunyli) [reply CHANGELOG.md]: Updated!

**Comment by kunyli** [comment]
@JoeRobich the tests are still failing, could you please help re-run or take a look at them?

**Comment by JoeRobich** [comment]
@kunyli We are now experiencing issues with downloading NuGet packages on our CI machines. Once that issue is resolved, I will be able to investigate your failure.

**Pull request merged**